docs/index.org

```diff
@@ -480,11 +480,15 @@ subject of our metadata. Our metadata will be keyed by the subject, so
 when querying a metadata service for the metadata of our asset, we
 will use the subject in our query.
 
-For phase-1 monetary scripts the standard for wallet metadata is that
+For tokens minted using native monetary policy the standard for metadata is that
```

Comment:
We need to agree on terminology but I believe _phase-1 monetary scripts_ is the correct one.

Reply:
Ok, rephrased in https://github.com/input-output-hk/offchain-metadata-tools/pull/38/commits/9e37bed17312a92d843e77946c34fdb62c2bc2b6.